pootle/apps/pootle_store/urls.py

```diff
@@ -29,9 +29,13 @@
         views.BrowseDataDispatcherView.as_view(),
         name='pootle-xhr-stats'),
 
-    url(r'^xhr/units/?$',
-        views.get_units,
-        name='pootle-xhr-units'),
+    url(r'^xhr/uids/?$',
+        views.get_uids,
+        name='pootle-xhr-uids'),
+
+    url(r'^xhr/view-rows/?$',
```

Comment:
`xhr/units/` felt a bit ambiguous (now that we have different API call to get the lightweight unit data). `xhr/view-rows/`, if seen in the XHR logs, tells exactly what data is being retrieved.